okta-hosted-login/src/test/java/com/okta/micronaut/example/ExampleApplicationTest.java

```diff
@@ -0,0 +1,21 @@
+package com.okta.micronaut.example;
+
+import io.micronaut.runtime.EmbeddedApplication;
+import io.micronaut.test.annotation.MicronautTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+
+import javax.inject.Inject;
+
+@MicronautTest
+public class ExampleApplicationTest {
+
+    @Inject
+    EmbeddedApplication application;
+
+    @Test
+    void testItWorks() {
+        Assertions.assertTrue(application.isRunning());
+    }
+
+}
```

Comment:
Does this pass if you run `mvn test`? A lot of times, it seems that when you add Okta, default tests don't pass.

Reply:
yes, the test runs successfully.